install-packages.sh

```diff
@@ -18,6 +18,17 @@ export DEBIAN_FRONTEND=noninteractive
 # Update the package listing, so we know what package exist:
 apt-get update
 
+#install universe so we can install more packages & update the package listing again
+apt-get install -y software-properties-common
+add-apt-repository universe          
+apt-get update
+
+#download & install oclint (no apt-get)
+apt-get install -y wget
+apt-get install -y tar
+wget https://github.com/oclint/oclint/releases/download/v21.03/oclint-21.03-llvm-11.1.0-x86_64-linux-ubuntu-20.04.tar.gz
+tar -xf oclint-21.03-llvm-11.1.0-x86_64-linux-ubuntu-20.04.tar.gz
```

Comment:
`install-packages.sh` is pretty generic.  It's also called twice from the `Dockerfile`.  I think this could be moved outside.  In the build section, where `python3` is installed, we could install `curl` and `tar`.

https://github.com/altendky/docker-ccstudio/blob/b90a16a1f3a5fbeb117a9f5755a92de3c1d3b16c/Dockerfile#L84-L86

Then `curl --location https://github.com/oclint/oclint/releases/download/v21.03/oclint-21.03-llvm-11.1.0-x86_64-linux-ubuntu-20.04.tar.gz | tar -xz` to go straight to an extracted directory.  Then copy that over to the final stage like CCStudio is.

https://github.com/altendky/docker-ccstudio/blob/b90a16a1f3a5fbeb117a9f5755a92de3c1d3b16c/Dockerfile#L94

Given the existing crazy size of this image, this kind of doesn't really matter.  But, maybe worth a little practice for both of us to see how it works out.